src/src/rx_main.cpp

```diff
@@ -620,7 +659,23 @@ void ICACHE_RAM_ATTR TXdoneISR()
 
 void setup()
 {
-    delay(100);
+    eeprom.Begin();
+    config.Load();
+
+#ifndef MY_UID
+    // Increment the power on counter in eeprom
+    config.SetPowerOnCounter(config.GetPowerOnCounter() + 1);
+    config.Commit();
+
+    // If we haven't reached our binding mode power cycles
+    // and we've been powered on for 2s, reset the power on counter
+    if (config.GetPowerOnCounter() < 3)
+    {
+        delay(2000);
+        config.SetPowerOnCounter(0);
+        config.Commit();
+    }
+#endif
```

Comment:
Should we add an `ifdef MY_UID` then `SetIsBound(true);`?

This would be a just in case someone later decides to call `GetIsBound`.

Reply:
No, I don't think this is a good idea. `SetIsBound()` should only be called after the UID bytes have been written to eeprom, which in this case they have not. I'd be more inclined to modify `GetIsBound()` so that it always returns true if `MY_UID` is defined.